### PR TITLE
Test case for lshw tool (Hardware lister)

### PR DESCRIPTION
**Pull request by ramyabs**
Description:
lshw is a small tool to provide detailed informaton on the hardware
configuration of the machine. It can report exact memory
configuration, firmware version, mainboard configuration, CPU version
and speed, cache configuration, bus speed, etc. on DMI-capable x86
systems and on some PowerPC machines (PowerMac G4 is known to work).

Signed-off-by: Ramya BS ramya@linux.vnet.ibm.com
Tested-by : Ramya BS <ramya@linux.vnet.ibm.com

### CODE REVIEW

File: generic/lshw.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        """
+        Outputs the device list showing bus information,
+        detailing SCSI, USB, IDE and PCI addresses.
+        """
+        bus_info = process.run("lshw -businfo")
+        if bus_info.exit_status:
+            self.fail(" lshw  failed to execute lshw -businfo  ")
+
+        # verifying the bus info for active network
+        get_bus = process.system_output('lshw -businfo').splitlines()
```
Comment (narasimhan-v) [inline generic/lshw.py]: `businfo.stdout` already has the output

**Comment by narasimhan-v** [comment]
Looks good, except for the one comment.

**Comment by ramyabs** [comment]
Closing this PR as have raised new PR https://github.com/avocado-framework-tests/avocado-misc-tests/pull/213  with the changes suggested .

**Comment by PraveenPenguin** [comment]
please hit close button as well :)

**Pull request closed without merging**